cwltool/sandboxjs.py

```diff
@@ -48,6 +68,14 @@ def new_js_proc():
             else:
                 raise
 
+    """ check nodejs version, if it is below certain threshold version,
+    raise Runtime Exception. Such a test won't be required for docker nodejs"""
+    if nodejs is not None:
+        if check_js_threshold_version(working_alias) is False:
+            raise JavascriptException(u"cwltool requires updated version of Node.js engine. "
```

Comment:
refactor out required version of node.js engine and share the required version with the user